### ISSUE TITLE
Start stream does not resume after a recording initiated by motion has ended. SD card filling up with files.

### ISSUE THREAD

**Issue by YapCK45** [s60sc/ESP32-CAM_MJPEG2SD#212]
V 8.6.3
1. **Start Stream** is initiated by clicking on the button and it changes to a blinking **Stop Stream** status. The display shows the current scene.
2. Motion detected starts capture and the **Start Recording** button begins to blink. When there is no more motion, recording ends and the status changes to a static **Start Recording**. The **Stop Stream** is still blinking but the display no longer shows the video. The status should change back to **Start Stream** as there is no longer any  motion.
3a. The **Start Recording** starts to blink even though there is no motion and this goes on indefinitely taking up storage space on the SD card. You can see the captured files on the SD card and there is no motion in them.
3b. If the **Start Recording** is clicked, it changes to **Stop Recording** and if it is clicked again, Recording apparently stops. But it starts again on its own without any motion and as in 3a, capturing files on the SD card without motion.
4. Eventually, this cycle stops or if the system is reset.

This observation can be replicated by having the web page on one half of the screen and the serial monitor on the other half of the screen as shown here:
![image](https://user-images.githubusercontent.com/64215797/233703901-63ed7318-1b5f-4a4e-86a3-ee250a5202ce.png)

The SD card is fast filling up as shown here:
![image](https://user-images.githubusercontent.com/64215797/233704238-a0f7b071-eac7-4de1-ace8-a0c55b366978.png)

**Comment by s60sc** [comment]
The stream freezes as the esp32 is busy recording, but only stops when requested by user.
I wasnt able to recreate the repeated motion issue.

**Comment by YapCK45** [comment]
Perhaps you could check on the recorded video files. I noticed that many of them did not have motion in them and I was wondering why. I would expected **every one** to start with motion unless they are initiated manually.

**Comment by YapCK45** [comment]
I want to elaborate on the earlier comment to look at the recorded files. Below is a collection of files captured on 20230423. I **playback the first three files** and noticed that they looked **exactly the same**. 
![image](https://user-images.githubusercontent.com/64215797/233829658-8c42edd3-b3d7-4286-83ef-56eb9bcefcfa.png)

At first, I thought this confirms my earlier comment. Then I suspected something is not right as there could be another possibility. So I **downloaded them** and found that the playback is different. So there is indeed motion capture.
Seems like the playback is not always playing back the file selected which led to my wrong conclusion. Apologies for that.

I did a ESP reboot and the playback is working as expected. Seems like somehow there is a issue in the file selection for playback. This is easily replicated but takes time to download the files for confirmation:

1. Select a few sequential files that are recorded for playback. If they show different scenes, then everything is working properly.
2. If they look the same, download them and playback to verify if there is motion and the scenes are different. If they are, it means the playback has some issue.
